README.md

```diff
@@ -11,95 +11,136 @@ Here are examples of using the inline scanner in different pipelines.
 *   [CircleCI](https://sysdig.com/blog/image-scanning-circleci/)
 
 ## Minimum Requirements
-* Sysdig Secure v2.5.0
-* Anchore Engine v0.5.0
-* Docker client access to pull images from Dockerhub
+* Sysdig Secure > v2.5.0 access (with token)
+* Docker client with daemon running
 * Internet Access to post results to Sysdig Secure
-* The latest version of the inline script will always be available at: https://download.sysdig.com/stable/inline_scan.sh
-* For on-prem Sysdig Secure customers the script will follow the convention below: 
-Sysdig Secure 3.2 - https://download.sysdig.com/stable/inline-scan-versioned/inline-scan-v3.2.sh
-
 
 **Note**: For Airgapped environments, we suggest the following:
-* docker pull docker.io/anchore/inline-scan:v0.6.1
+
+* docker pull docker.io/anchore/inline-scan:v0.6.1 (the version may differ based on the installation)
 * docker pull sysdiglabs/secure-inline-scan:latest (if using the inline scan container)
 * Open firewall settings to allow traffic to https://secure.sysdig.com/api/scanning
 
 **Note**: For onprem environments, use the -o flag in order to get the correct scan result URL.
 
 ## Usage
 
-Simple one line option 
+Here below are described different usages based on Sysdig installation type.
 
-    $ curl -s https://download.sysdig.com/stable/inline_scan.sh | bash -s -- analyze -s https://secure.sysdig.com -k <ACCESS_KEY> -P <FULL_IMAGE_TAG>
+### OnPrem
```

Comment:
super nit: do we want to show the SaaS usage first and then the Onprem instructions, just for visibility

Reply:
Since nobody else commented I suppose it's not necessary 😅